### PR TITLE
Split apart auth schemas

### PR DESCRIPTION
**Pull request by kaysoky**
- May lower the amount of confusion over which fields are required for which APIs
- Also adds java client method for GET /storageUsage

### CODE REVIEW

File: lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/auth/Username.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"description": "Used to login via username and password",
```
Comment (kaysoky) [inline lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/auth/Username.json:2]: Fix this (no copy-paste :)

File: lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/auth/UsernameInterface.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+	"description": "Contains the user's email and password",
```
Comment (kaysoky) [inline lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/auth/UsernameInterface.json:2]: Fix this (no copy-paste :)

File: lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/auth/LoginCredentials.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+	"description": "Used to login via username and password",
```
Comment (brucehoff) [inline lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/auth/LoginCredentials.json:2]: add to description: "allowing client also to specify that the Synapse terms of use are accepted by the user."

File: services/repository/src/main/java/org/sagebionetworks/auth/AuthenticationController.java [hunk 4]
```diff
@@ -104,18 +102,16 @@ public void createUser(@RequestBody NewUser user) throws NotFoundException {
...
 	@RequestMapping(value = UrlHelpers.AUTH_REGISTERING_USER_EMAIL, method = RequestMethod.POST)
-	public void resendRegisteringUserPasswordEmail(@RequestBody NewUser user) throws NotFoundException {
+	public void resendRegisteringUserPasswordEmail(@RequestBody Username user) throws NotFoundException {
 		authenticationService.sendUserPasswordEmail(user.getEmail(), PW_MODE.SET_PW);
 	}
 	
 	/**
 	 * Retrieve basic information about the current authenticated user.  
-	 * Information includes the user's display name, email, and whether they have accepted the terms of use.
+	 * Information includes the user's display name, and email.
```
Comment (kaysoky) [inline services/repository/src/main/java/org/sagebionetworks/auth/AuthenticationController.java:114]: extra comma

**Pull request merged**